clients/js/src/digitalAsset.ts

```diff
@@ -115,6 +118,32 @@ export async function fetchAllDigitalAssetByCreator(
   return fetchAllDigitalAsset(context, mints, options);
 }
 
+/**
+ * Fetches all digital assets from a verified collection. This does not work on older nfts that do not have a tokenStandard set.
+ */
+export async function fetchAllDigitalAssetByVerifiedCollection(
+  context: Pick<Context, 'rpc' | 'eddsa' | 'programs'>,
+  collectionAddress: PublicKey,
+  options?: RpcGetAccountsOptions
+): Promise<DigitalAsset[]> {
+  const mints = await Promise.all(
+    COLLECTION_OFFSETS.map(async (offset) => {
+      try {
+        return await getMetadataGpaBuilder(context)
+          .where(offset, 1)
+          .where(offset + VERIFIED_COLLECTION_OFFSET, 1)
+          .where(offset + COLLECTION_ADDRESS_OFFSET, collectionAddress)
+          .sliceField('mint')
+          .getDataAsPublicKeys();
+      } catch (e) {
+        return [];
```

Comment:
Do we need to bubble the error to the client? E.g., if there is a connection error.

Reply:
Yea good point, i've removed the try / catch